app/views/unattended/provisioning_templates/registration/global_registration.erb

```diff
@@ -57,7 +62,7 @@ if [ x$ID = xrhel ] || [ x$ID = xcentos ]; then
     yum localinstall $CONSUMER_RPM -y
     rm -f $CONSUMER_RPM
 
-    subscription-manager register --org=<%= @organization.label %> --activationkey="sample-key"
+    subscription-manager register --org=<%= @organization.label %> --activationkey="<%= @activation_key %>"
```

Comment:
Can we detect if the katello plugin is actually available?

Reply:
Yes, we do that: https://github.com/theforeman/foreman/blob/a2c96913c1873dfe00630a6c097ac5fea4823e6f/app/views/unattended/provisioning_templates/registration/global_registration.erb#L48